IanGRap/Squish
# Fix audio fade bug #116

Contributors: Alfredo Cuevas

Goal: Fix Issue #116 Where two separate audio sources were not able to fade properly at the same time

Changes: 
* AudioManager.cs: Created two dictionaries that replace the global fading booleans.  The dictionaries track the fading status of every audio source created on Awake
* InputReaderTest.cs: A new Temporary script created for the purposes of testing the changes
* AudioTest.unity: A temporary scene created to test the changes

How to Test: 
Go into the AudioTest scene
Press play. 
The Up arrow key calls FadeIn on the "Gameplay Theme". Down Arrow calls FadeOut
The Right arrow key calls the FadeIn on the "Menu Theme". Left Arrow calls FadeOut.
* Check to make sure that both songs can fade in / fade out at the same time.
* Play with the keys for a minute and see if you can break anything. 
* Note: For the test the fade In/Out timers were set to 10 seconds.
